bpf/init.sh

```diff
@@ -512,7 +512,6 @@ if [ "$MODE" = "vxlan" -o "$MODE" = "geneve" ]; then
 	ip link set $ENCAP_DEV mtu $MTU || encap_fail
 
 	setup_dev $ENCAP_DEV
-	ip link set $ENCAP_DEV up || encap_fail
```

Comment:
Would it make sense to add the `|| encap_fail` part above on the `ip link set ... up` part of `setup_dev`?

Reply:
Makes sense to me. Fixed.